src/courseware/course/sequence/Sequence.test.jsx

```diff
@@ -419,3 +430,30 @@ describe('Sequence', () => {
     });
   });
 });
+
+describe('window.addEventListener', () => {
+  it('calls sidebarNavigationClickHandler with the correct argument', () => {
+    global.window = Object.create(window);
+    const sidebarNavigationClickHandler = jest.fn();
+
+    // Add Event listener
+    window.addEventListener('message', event => {
+      if (event.data.message === 'outline_sidebar_navigation_started') {
+        sidebarNavigationClickHandler(event.data.subsection_id);
+      }
+    });
+
+    const event = {
+      data: {
+        message: 'outline_sidebar_navigation_started',
+        subsection_id: 'xBlockSubsectionId',
+      },
+    };
```

Comment:
Please, add a blank lien after this line to separate the assignation and the act sections. @anfbermudezme